pkg/plugins/manager/signature/manifest.go

```diff
@@ -169,6 +169,7 @@ func (s *Signature) Calculate(ctx context.Context, src plugins.PluginSource, plu
 
 	// Make sure the versions all match
 	if manifest.Plugin != plugin.JSONData.ID || manifest.Version != plugin.JSONData.Info.Version {
+		s.log.Debug("Plugin signature invalid because ID or Version mismatch", "id", plugin.JSONData.ID, "manifest", manifest.Plugin, "version", manifest.Version, "plugin", plugin.JSONData.ID, "version", plugin.JSONData.Info.Version)
```

Comment:
```suggestion
		s.log.Debug("Plugin signature invalid because ID or Version mismatch", "pluginId", plugin.JSONData.ID, "manifest", manifest.Plugin, "version", manifest.Version, "pluginId", plugin.JSONData.ID, "version", plugin.JSONData.Info.Version)
```